src/LiveComponent/assets/package.json

```diff
@@ -1,16 +1,17 @@
 {
     "name": "@symfony/live-stimulus",
     "description": "Live Component: bring server-side re-rendering & model binding to any element.",
-    "main": "dist/live_controller.js",
+    "main": "dist/live-controller.esm.js",
```

Comment:
Do you think it would be interesting to also configure `module` field?

Reply:
Even though they're identical, yea, I think that's a good idea.